src/date.html

```diff
@@ -0,0 +1,61 @@
+<div class="form-group schema-form-{{form.type}} {{form.htmlClass}}"
+     ng-class="{'has-error': form.disableErrorState !== true && hasError(), 'has-success': form.disableSuccessState !== true && hasSuccess(), 'has-feedback': form.feedback !== false }">
+  <label class="control-label {{form.labelHtmlClass}}" ng-class="{'sr-only': !showTitle()}" for="{{form.key.slice(-1)[0]}}">{{form.title}}</label>
+
+  <div id="{{form.key.slice(-1)[0]}}"
+       ng-show="form.key"
+       rds-date
+       sf-field-model="replaceAll"
+       ng-init="initInternalModel($$value$$);"
+       ng-model="$$value$$">
+    <input type="date" ng-model="dateCtrl.date" class="form-control {{form.fieldHtmlClass}}" ng-disabled="form.readonly" sf-changed="form" schema-validate="form">
+  </div>
+
+  <!--<input ng-if="!form.fieldAddonLeft && !form.fieldAddonRight"-->
```

Comment:
I left this stuff here because the wrapped date currently makes arrays validate incorrectly and something here probably fixes it... not sure why, but it's not blocking us atm.